src/widgetastic_patternfly4/dropdown.py

```diff
@@ -187,6 +188,11 @@ def item_select(self, item, handle_alert=None, **kwargs):
                 self.logger.warning("There is an unexpected alert present.")
                 pass
 
+    @property
+    def selected_item(self):
```

Comment:
Why is it here? According to https://patternfly-react.surge.sh/patternfly-4/documentation/react/components/dropdown Dropdown doesn't keep selected item.

Reply:
Yes true @skarekrow the selected item stuff only needs to live in the "Select" class